operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -164,7 +165,13 @@ func CreateOrUpdateLokiStack(
 	}
 
 	objects, err := manifests.BuildAll(opts)
-	if err != nil {
+	if errors.Is(err, lokiv1.ErrReplicationFactorTooHigh) {
+		return nil, &status.DegradedError{
```

Comment:
We can't return the status.DegradedErrror in `operator/internal/manifests/ingester.go` because of an import cycle, right? I wonder if we should just move the definition of `status.DegradedError` around to allow package `manifests` to use it. 

cc @xperimental any thoughts?